src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -312,4 +312,7 @@
   <data name="FormatDateTimeOffset" xml:space="preserve">
     <value>The JSON value is of unsupported format for a DateTimeOffset.</value>
   </data>
+  <data name="FormatGuid" xml:space="preserve">
+    <value>The JSON value is of unsupported format for a Guid.</value>
```

Comment:
Nit: this text isn't grammatically correct; same for the existing strings of the same format.